### ISSUE TITLE
Lutris ScummVM runner: miss option to change language

### ISSUE THREAD

**Issue by legluondunet** [legluondunet/MyLittleLutrisScripts#34]
Sorry for the delay, finally got around to testing it, great step forward, but now the error from logs is: 
```
Running /home/jeff/.local/share/lutris/runners/scummvm/bin/scummvm --extrapath=/home/jeff/.local/share/lutris/runners/scummvm/share/scummvm --themepath=/home/jeff/.local/share/lutris/runners/scummvm/share/scummvm --aspect-ratio --fullscreen --gfx-mode=3x --path=/home/jeff/Recreation/Games/Lutris/myst-masterpiece-edition myst-win
Initial process has started with pid 694277
Game is considered started.
scummvm: Unrecognized game target 'myst-win'
Usage: scummvm [OPTIONS]... [GAME]

Try 'scummvm --help' for more options.
Game is considered exited.
Initial process has exited.
All children have exited.
Exit with returncode 256
```

The contents of the installation directory (not empty this time, yay!) are:
```
╭─ ~/R/Games/Lutris/myst-masterpiece-edition
╰─ ls -l
total 355632
-rw-r--r--  1 jeff jeff 63780140 Apr  3  2018 channel.dat
-rw-r--r--  1 jeff jeff    28758 Jul 30  2018 channel_french.dat
-rw-r--r--  1 jeff jeff    26560 Jul 30  2018 channel_german.dat
-rw-r--r--  1 jeff jeff    23186 Jul 30  2018 channel_polish.dat
-rw-r--r--  1 jeff jeff    11076 Jul 30  2018 channel_spanish.dat
-rw-r--r--  1 jeff jeff  4870667 Apr  3  2018 credits.dat
-rw-r--r--  1 jeff jeff    80722 Jul 30  2018 credits_french.dat
-rw-r--r--  1 jeff jeff    82336 Jul 30  2018 credits_german.dat
-rw-r--r--  1 jeff jeff  8796556 Apr  3  2018 dunny.dat
drwxr-xr-x  3 jeff jeff     4096 Apr 21 00:14 french
drwxr-xr-x  3 jeff jeff     4096 Apr 21 00:14 german
-rw-r--r--  1 jeff jeff  1849077 Apr  3  2018 help.dat
-rw-r--r--  1 jeff jeff  1864898 Jul 30  2018 help_french.dat
-rw-r--r--  1 jeff jeff  1923844 Jul 30  2018 help_german.dat
-rw-r--r--  1 jeff jeff  2809658 Jul 30  2018 help_polish.dat
-rw-r--r--  1 jeff jeff  3161356 Apr  3  2018 intro.dat
-rw-r--r--  1 jeff jeff     4561 Jun 26  2018 LICENSE.NOTO.txt
-rw-r--r--  1 jeff jeff   592448 Apr  3  2018 manual.pdf
-rw-r--r--  1 jeff jeff 33073173 Apr  3  2018 mechan.dat
-rw-r--r--  1 jeff jeff   133862 Jul 30  2018 mechan_french.dat
-rw-r--r--  1 jeff jeff   136060 Jul 30  2018 mechan_german.dat
-rw-r--r--  1 jeff jeff   134658 Jul 30  2018 mechan_polish.dat
-rw-r--r--  1 jeff jeff    44656 Jul 30  2018 mechan_spanish.dat
-rw-r--r--  1 jeff jeff   284830 Jun 26  2018 menu.dat
-rw-r--r--  1 jeff jeff 74070233 Apr  3  2018 myst.dat
-rw-r--r--  1 jeff jeff  6549364 Jul 30  2018 myst_french.dat
-rw-r--r--  1 jeff jeff  6652814 Jul 30  2018 myst_german.dat
-rw-r--r--  1 jeff jeff  6723446 Jul 30  2018 myst_polish.dat
-rw-r--r--  1 jeff jeff  2571548 Jul 30  2018 myst_spanish.dat
-rw-r--r--  1 jeff jeff   314640 Jun 26  2018 NotoSans-ExtraBold.ttf
drwxr-xr-x  3 jeff jeff     4096 Apr 21 00:14 polish
drwxr-xr-x 10 jeff jeff     4096 Apr 21 00:15 qtw
-rw-r--r--  1 jeff jeff 69538999 Apr  3  2018 selen.dat
-rw-r--r--  1 jeff jeff  2452425 Jul 30  2018 selen_french.dat
-rw-r--r--  1 jeff jeff   349160 Jul 30  2018 selen_polish.dat
-rw-r--r--  1 jeff jeff    61110 Jul 30  2018 selen_spanish.dat
drwxr-xr-x  3 jeff jeff     4096 Apr 21 00:14 spanish
-rw-r--r--  1 jeff jeff 69459412 Apr  3  2018 stone.dat
-rw-r--r--  1 jeff jeff  1250240 Jul 30  2018 stone_french.dat
-rw-r--r--  1 jeff jeff   145814 Jul 30  2018 stone_german.dat
-rw-r--r--  1 jeff jeff   147880 Jul 30  2018 stone_polish.dat
-rw-r--r--  1 jeff jeff    62126 Jul 30  2018 stone_spanish.dat
```

_Originally posted by @jeffvandyke in https://github.com/legluondunet/MyLittleLutrisScripts/issues/30#issuecomment-616981654_

**Comment by legluondunet** [comment]
In Lutris ScummVM we can not precise language option and a lot of more options available here:
https://wiki.scummvm.org/index.php/User_Manual/Appendix:_Command_line_options
The idea is to permit to advanced users  to add custom command lines options to ScummVM.
Some scripts I wrote are affected by this issue: Myst Masterpiece edition and Riven.

**Comment by legluondunet** [comment]
Lutris ScummVM runner updated with this functionality

**Comment by legluondunet** [comment]
@jeffvandyke I will soon updated my scripts

**Comment by legluondunet** [comment]
Scripts updated with new Lutris ScummVM args option to choose language:
Myst
Riven
BASS
This new option will be available in next Lutris version (> 0.5.6).
When you install this game with actual Lutris version, the default language used will be english, no matter the language you choosed during install.
Issue closed, thank you for your report.

**Comment by jeffvandyke** [comment]
Thanks so much, runs like a charm now; I'm looking forward to exploring this game!
